db/models/song.js

```diff
@@ -0,0 +1,66 @@
+var mongoose = require('mongoose');
+
```

Comment:
I think there should be a connection to the database made, such as: mongoose.connection.  And if so, this connection should be exported such that the connection can be used to access the db.